src/main/java/teammates/storage/api/FeedbackResponsesDb.java

```diff
@@ -78,17 +71,13 @@ public FeedbackResponseAttributes getFeedbackResponse(
      * @return Null if not found.
      */
     public FeedbackResponse getFeedbackResponseEntityWithCheck(String feedbackResponseId) {
-
```

Comment:
To record our discussion: won't be changing this as I'd like to keep the whitespace consistent throughout the Storage layer.